Structure_Engine/Query/InternalElements2D.cs

```diff
@@ -32,16 +35,24 @@ public static partial class Query
         /****               Public Methods              ****/
         /***************************************************/
 
+        [Description("Gets inner IElement2Ds from an Opening. Method required for all IElement2Ds. \n" + 
+                     "As an opening does not contain any internal elements, this will always return an empty list.")]
+        [Input("opening","The Opening to get internal IElement2Ds from.")]
+        [Output("elements", "Inner elements. This will always be an empty list for openings.")]
         public static List<IElement2D> InternalElements2D(this Opening opening)
         {
             return new List<IElement2D>();
         }
 
         /***************************************************/
```

Comment:
What's the point of this method if it is just returning a new `List<IElement2D>()`?

Reply:
Required method for all IElement2Ds. Both Panel and Opening are of this type.

Reply:
I think the description is enough.